multiversx_sdk_cli/tests/test_cli_transactions.py

```diff
@@ -105,5 +107,62 @@ def test_create_multi_transfer_transaction_with_single_egld_transfer(capsys: Any
     assert data == "MultiESDTNFTTransfer@8049d639e5a6980d1cd2392abcce41029cda74a1563523a202f09641cc2618f8@01@45474c442d303030303030@@0de0b6b3a7640000"
 
 
+def test_create_relayed_v3_transaction(capsys: Any):
+    return_code = main([
+        "tx", "new",
```

Comment:
Not possible to take a user transaction, already signed, and sign it with a relayer? Maybe `mxpy tx relay` or `mxpy wallet sign-as-relayer`? :thought_balloon: 

I think we should allow users to decouple (in a way or another) the step of signing as user from the step of signing as relayer.

Maybe brainstorm this a bit?